docs/superpowers/specs/2026-06-06-yutori-scouting-design.md

```
@@ -0,0 +1,489 @@
+We do not implement HMAC signature verification because Yutori's
+documentation does not describe a signing scheme. If Yutori later
+publishes one, swap in HMAC and drop the token-in-path requirement.
+
+CSRF: the view is decorated with `@csrf_exempt` (it's an external webhook).
+
+## Data flow — detail
+
+### One-time setup
+
+```text
+$ python manage.py migrate scout
```

markdownlint-cli2 (0.22.1)

[warning] Dollar signs used before commands without showing output

(MD014, commands-show-output)